gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/plugin/domain_service/PluginFilterByLicenseDomainService.java

```diff
@@ -28,7 +29,15 @@ public PluginFilterByLicenseDomainService(LicenseManager licenseManager) {
         this.licenseManager = licenseManager;
     }
 
-    public Set<ConnectorPlugin> setPluginDeployedStatusDependingOnLicense(Set<ConnectorPlugin> pluginSet, String organizationId) {
+    public Set<ConnectorPlugin> setConnectorPluginDeployedStatusDependingOnLicense(Set<ConnectorPlugin> pluginSet, String organizationId) {
```

Comment:
Maybe the object name could be more generalized, like `Plugin`? Could be that I don't properly understand what the policies do, but they seem more like `plugins` rather than `connector plugins` (such as entrypoint or endpoint).